### ISSUE TITLE
py27-lint: commands failed

### ISSUE THREAD

**Issue by HadleyKing** [biocompute-objects/galaxy#15]
From https://github.com/galaxyproject/galaxy/pull/9077

https://github.com/biocompute-objects/galaxy/blob/2148e976a8f78da3ec21febe0ce8e22361c37808/lib/galaxy/webapps/galaxy/api/histories.py#L571

```bash
#!/bin/bash -eo pipefail
tox -e py27-lint
py27-lint create: /home/circleci/repo/.tox/py27-lint
py27-lint installdeps: -rlib/galaxy/dependencies/pipfiles/flake8/pinned-requirements.txt
py27-lint installed: DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support,configparser==4.0.2,entrypoints==0.3,enum34==1.1.6,flake8==3.7.9,flake8-import-order==0.18.1,functools32==3.2.3.post2,mccabe==0.6.1,pycodestyle==2.5.0,pyflakes==2.1.1,typing==3.7.4.1
py27-lint run-test-pre: PYTHONHASHSEED='1570103925'
py27-lint run-test: commands[0] | bash .ci/flake8_wrapper.sh
./lib/galaxy/webapps/galaxy/api/histories.py:571:17: E265 block comment should start with '# '
ERROR: InvocationError for command /bin/bash .ci/flake8_wrapper.sh (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py27-lint: commands failed

Exited with code exit status 1
```

Also py35-lint for the same thing.
```bash
#!/bin/bash -eo pipefail
tox -e py35-lint
py35-lint create: /home/circleci/repo/.tox/py35-lint
py35-lint installdeps: -rlib/galaxy/dependencies/pipfiles/flake8/pinned-requirements.txt
py35-lint installed: entrypoints==0.3,flake8==3.7.9,flake8-import-order==0.18.1,mccabe==0.6.1,pycodestyle==2.5.0,pyflakes==2.1.1
py35-lint run-test-pre: PYTHONHASHSEED='3139246627'
py35-lint run-test: commands[0] | bash .ci/flake8_wrapper.sh
./lib/galaxy/webapps/galaxy/api/histories.py:571:17: E265 block comment should start with '# '
ERROR: InvocationError for command /bin/bash .ci/flake8_wrapper.sh (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py35-lint: commands failed

Exited with code exit status 1
```

**Comment by HadleyKing** [comment]
@corburn 
Can/should I just uncomment that line?

**Comment by corburn** [comment]
@HadleyKing the lint warning is complaining because there is not a space after the '#':

```python
 # 'input_subdomain': [], 
```

**Comment by corburn** [comment]
resolved by c2de07cfd047b429497ceb81fac163f4dc04d5b0
